docusaurus/src/theme/EditThisPage/index.tsx

```diff
@@ -0,0 +1,24 @@
+// Swizzle action eject is unsafe to perform on EditThisPage.
+// It is more likely to be affected by breaking changes in the future
+// https://docusaurus.io/docs/swizzling/
+//
+// We added nofollow to the link for SEO purposes.
+import React from 'react';
+import Translate from '@docusaurus/Translate';
+import {ThemeClassNames} from '@docusaurus/theme-common';
+import Link from '@docusaurus/Link';
+import IconEdit from '@theme/Icon/Edit';
+import type {Props} from '@theme/EditThisPage';
+
+export default function EditThisPage({editUrl}: Props): JSX.Element {
+  return (
+    <Link to={editUrl} className={ThemeClassNames.common.editThisPage} rel="nofollow noopener noreferrer">
```

Comment:
suggestion

I see that the current link on our docs has also `target=_blank`. Will it stay the same? Does `Link` take care of that internally?

Reply:
> I see that the current link on our docs has also target=_blank. Will it stay the same?

Yes.

> Does Link take care of that internally?

Yes.